test/functional/test_framework/test_node.py

```diff
@@ -358,7 +358,7 @@ def is_node_stopped(self):
         return True
 
     def wait_until_stopped(self, timeout=BITCOIND_PROC_WAIT_TIMEOUT):
-        wait_until(self.is_node_stopped, timeout=timeout, timeout_factor=self.timeout_factor)
+        wait_until(self.is_node_stopped, timeout=timeout)
```

Comment:
Do you need to multiply `timeout` by `timeout_factor` here?

Reply:
Timeout here is set to `BITCOIND_PROC_WAIT_TIMEOUT` which is hardcode as 60 secs. This is the only time this value is being called. As it has been specifically separated from `self.options.timeout`, would it make sense to multiply it with `timeout_factor` here? Thinking that I skipped it.

Reply:
Polite reminder. :) 
Does the above rationale make sense or should I change it?

Reply:
> Timeout here is set to BITCOIND_PROC_WAIT_TIMEOUT which is hardcode as 60 secs.

That's the default value. `wait_until_stopped()` can be called with any timeout.

Reply:
Right that was just a default. agreed and fixed.